.gitignore

```diff
@@ -22,3 +22,6 @@ pkg/i18n/translations.go
 _logs/
 test/acs-engine-test/report/TestReport.json
 *.swp
+
+# I have no idea why these get generated when I run the e2e test
+test/e2e/kubernetes/translations/ 
```

Comment:
as far as i can tell yes they are generated by running something that is part of the e2e tests (maybe the az cli does it?). it was annoying me that i kept having to remove them so figured i would just add it to the gitignore